crates/wasi/src/instantiate.rs

```diff
@@ -9,20 +9,19 @@ use std::fs::File;
 use std::rc::Rc;
 use target_lexicon::HOST;
 use wasi_common::{WasiCtx, WasiCtxBuilder};
-use wasmtime_api as api;
 use wasmtime_environ::{translate_signature, Export, Module};
 use wasmtime_runtime::{Imports, InstanceHandle, InstantiationError, VMFunctionBody};
 
-/// Creates `api::Instance` object implementing the "wasi" interface.
+/// Creates `wasmtime::Instance` object implementing the "wasi" interface.
 pub fn create_wasi_instance(
-    store: &api::HostRef<api::Store>,
+    store: &wasmtime::HostRef<wasmtime::Store>,
     preopened_dirs: &[(String, File)],
     argv: &[String],
     environ: &[(String, String)],
-) -> Result<api::Instance, InstantiationError> {
+) -> Result<wasmtime::Instance, InstantiationError> {
     let global_exports = store.borrow().global_exports().clone();
     let wasi = instantiate_wasi(global_exports, preopened_dirs, argv, environ)?;
-    let instance = api::Instance::from_handle(&store, wasi);
+    let instance = wasmtime::Instance::from_handle(&store, wasi);
```

Comment:
If you rebase to the latest `master` and apply the same changes to `crates/wasi/src/old/snapshot_0/instantiate.rs`, the CI should work just fine. I don't think there is anything wrong with `witx` as such.

Reply:
The build never got this far, which is why I didn't see this. Thank you for catching this, I'll update.

I don't think this relates to the issue with `witx`, which seems unrelated:

```
error[E0599]: no method named `comment` found for type `wast::parser::Cursor<'_>` in the current scope
   --> crates/wasi-common/WASI/tools/witx/src/parser.rs:111:49
    |
111 |                 let (comment, c) = match cursor.comment() {
    |                                                 ^^^^^^^ method not found in `wast::parser::Cursor<'_>`
```

Reply:
Ah, I see. Somehow I missed that error, thanks for pointing it out! :-)